firestore.rules

```diff
@@ -0,0 +1,7 @@
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read: if true;
```

Comment:
**question**: is it ok that any document can be read by anyone?

Reply:
It's not a big deal. Although, we will have some emails in there at some point. I actually don't need anything to be able to read this data other than the cloud run service. Is there a better way to achieve this?